build-system/tasks/presubmit-checks.js

```diff
@@ -15,6 +15,9 @@
  */
 'use strict';
 
+// Terms are defined here.
+/* eslint-disable local/terms */
```

Comment:
With the rule enabled, [the errors generated are a good example for the linter.](https://gist.github.com/alanorozco/f311e9cbe39d70e7d6dccb54f7c73f1b)